preprocessors/openstreetmap/osm_service.py

```diff
@@ -82,16 +93,16 @@ def get_streets(bbox_coord):
     except Exception:
         try:
             error = 'Primary server not responding, so connecting server 1'
```

Comment:
Suggest logging the actual server names rather than just "primary", "server 1", "server 2". The `config.py` may change (or be updated in the future so servers can be overridden) and these messages will not be nearly as informative.